conf/messages

```diff
@@ -26,24 +26,35 @@ otpafe.site.error.page.header = There are errors on the page.
 add.page.title = Add Entry
 add.page.header = Add Entry
 add.page.add = Add
-add.page.action = What are you trying to do?
-add.page.nino = National Insurance Number
-add.page.nino.hint = Optional if SA UTR provided
-add.page.sautr = Self Assessment Unique Tax Reference
-add.page.sautr.hint = Optional if NINO provided
-add.page.group = Group
-add.page.group.hint = Optional way to tag this entry
-add.page.identityProvider = Identity Provider
-add.page.identityProviderId = Identity Provider Id
-add.page.addedByTeam = Added by team
-add.page.action.select.LOCK = Lock a NINO or SA UTR to the credential ID provided.
-add.page.action.select.BLOCK = Block a NINO or SA UTR from being used.
+add.submit.button = Add
```

Comment:
there needs to be welsh for all this added text

Reply:
not required for this app